internal/kubernetes/operator/features/mock.go

```diff
@@ -0,0 +1,7 @@
+package features
+
+type MockFeatureValidator struct{}
+
+func (*MockFeatureValidator) IsFeatureExist(_, _ string) bool {
+	return true
+}
```

Comment:
@Sugar-pack recently moved all the operator mocks to autogenerated mocks, I think the team needs to pick a single approach

Reply:
Fixed